frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/object-service-dashboards.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { SECOND } from '../../integration-tests/utils/consts';
+
+const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}`;
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: 'openshift-storage',
+    name: testName,
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const installStorageCluster = () => {
+  cy.login();
+  cy.visit('/');
+  cy.install();
+};
+
+describe('Tests Buckets Card', () => {
+  before(() => {
+    installStorageCluster();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.byLegacyTestID('horizontal-link-Object Service').click();
+  });
+
+  it('Test at least one noobaa bucket is present', () => {
+    cy.wait(10 * SECOND)
+      .byTestID('buckets-status-title')
+      .invoke('text')
+      .then((text) => {
+        const count = parseInt(text.split(' ')[0], 10);
+        expect(count).toBeGreaterThanOrEqual(1);
+      });
+  });
+
+  it('Create an Object Bucket Claim and test equality', () => {
+    cy.wait(10 * SECOND)
```

Comment:
As per best practices, arbitrary wait times are not allowed. This is one of the issues i faced in my PR, you probably need to look into this

Reply:
Thanks for pointing this out! Sending a patch soon based on [this](https://docs.cypress.io/guides/references/best-practices.html#Unnecessary-wait-for-cy-get) soon.

Reply:
It seems the above method is only for routes. Using `{ timeout: __ }` instead.